jasmine/src/game.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+class Game {
+  constructor(runningScore, bowlCard, scoreCard, index) {
+    this.runningScore = 0;
+    this.bowlCard = [];
+    this.scoreCard = [];
+    this.index = -1;
+  }
+
+  roll(bowl1, bowl2 = 0) {
+    this.index += 1;
+    this.bowlCard.push([bowl1, bowl2]);
+    this.scoreCard.push(bowl1 + bowl2);
+    this.runningScore += (bowl1 + bowl2);
+    this.calculateScore();
+    return bowl1 + bowl2;
+  }
+ 
+  calculateSparePoints() {
+    let sparePoints = this.bowlCard[this.index][0];
+    if (this.scoreCard[this.index-1] === 10 && this.bowlCard[this.index-1][0] !== 10) {
+      this.scoreCard[this.index-1] += sparePoints;
+      this.runningScore += sparePoints;
+    } return;
+  }
```

Comment:
Methods like this can be hard to read, because it's not immediately obvious what something like `this.bowlCard[this.index-1][0]` refers to. It would be good to name these with variables to make it easier to read, eg
```js
const previousFrameRolls = this.bowlCard[this.index-1]
const previousFrameTotal = this.scoreCard[this.index-1]
if (previousFrameTotal === 10 && previousFrameRolls[0] !== 10) {...}
```